packages/run-protocol/test/attestation/test-attestation.js

```diff
@@ -60,6 +60,7 @@ const makeContext = async t => {
   const uBrand = stakeKit.brand;
 
   const { publicFacet, creatorFacet } = await makeAttestationFacets(
+    // @ts-expect-error cast
```

Comment:
I don't think this cast is necessary. `makeAttestationFacets` should accept any zcf.

Reply:
Got it.